lib/openstudio-standards/utilities/assertion.rb

```diff
@@ -0,0 +1,17 @@
+# PRM generation custom Error
+class PRMError < StandardError
+end
+
+# Finds capacity in W
```

Comment:
Is that a place holder? The description doesn't seem to match with what's in the function.

Reply:
It was a copy paste error - thanks for pointing it out!